### DISCUSSION TITLE
[Accessory Request] WiFi Vacuum Cleaners

### DISCUSSION THREAD

**Discussion by blloyd78** [milo526/homebridge-tuya-web#224] [Q&A] [answered]
I appreciate there are several plugins for Neato/Xiaomi/Roborock vacuum cleaners available but none of these can interface correctly with the vacuum cleaners registered in Tuya - like models from Kyvol/Viomi etc.

Any chance we can get vacuum support added to this plugin? Even basic functionality like switch on and return to base would be enough for a start...

Thanks in advance.

**Comment by milo526** [comment, marked as answer]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Reply by blloyd78** [reply]
Thanks Milo. I have run the script as instructed but the Kyvol vacuum cleaner is not listed in the output.

Guess I'm out of luck.

Thanks for your help anyway.
